android/capacitor/src/main/java/com/getcapacitor/plugin/notification/LocalNotificationManager.java

```diff
@@ -174,6 +175,17 @@ private void buildNotification(NotificationManagerCompat notificationManager, Lo
     mBuilder.setOnlyAlertOnce(true);
 
     mBuilder.setSmallIcon(localNotification.getSmallIcon(context));
+
+    String iconColor = localNotification.getIconColor();
+    if (iconColor != null) {
+      try {
+        mBuilder.setColor(Color.parseColor(iconColor));
+      } catch (Exception ex) {
+        call.error("The iconColor string was not able to be parsed.  Please provide a valid string hexidecimal color code.", ex);
```

Comment:
For consistency, better use the same error message as Status Bar plugin

```suggestion
        call.error("Invalid color provided. Must be a hex string (ex: #ff0000");
```

Reply:
fixed